### ISSUE TITLE
rest api: fix filters for text/csv

### ISSUE THREAD

**Issue by ChHaeni** [oposs/agrammon#549]
Currently, the option to include filters in the model results is only provided for application/json output.

Add the results for individual animal categories (`include-filters=true`) to csv output (`text/csv`), too.

Possible formats could be:
  - one row per filter, as in the CLI output, where the column 'filter' contains the value of the animal category and the column 'value' the corresponding values
  - add a secondary separator (e.g. ";") and write all filters and all corresponding values separated by this secondary separator as one entry in columns 'filter' and 'value', respectively

**Comment by zaucker** [comment]
PR #562 implements this format:

```
##  Model: version6/End.nhd
##  Variants: Base
### Simulation Q2015_Version6_20210818
##  Print filter: TotalNH3
#   Dataset 10004
Q2015_Version6_20210818;10004;Total;nh3_nanimalproduction;total;1993.3358358888165;kg N/Jahr
Q2015_Version6_20210818;10004;Total;nh3_nanimalproduction;suckling_cows;0;kg N/Jahr
Q2015_Version6_20210818;10004;Total;nh3_nanimalproduction;heifers_1st_yr;0;kg N/Jahr
Q2015_Version6_20210818;10004;Total;nh3_nanimalproduction;heifers_2nd_yr;0;kg N/Jahr
Q2015_Version6_20210818;10004;Total;nh3_nanimalproduction;heifers_3rd_yr;1398.4691299197375;kg N/Jahr
Q2015_Version6_20210818;10004;Total;nh3_nanimalproduction;fattening_calves;0;kg N/Jahr
Q2015_Version6_20210818;10004;Total;nh3_nanimalproduction;calves_suckling_cows;0;kg N/Jahr
Q2015_Version6_20210818;10004;Total;nh3_nanimalproduction;beef_cattle;594.866705969079;kg N/Jahr
```
The pseudo variable name `total` is used for the sum of the filtered values.

Currently variable names are output. Alternatively, the translated label could be used instead or in addition.

**Comment by zaucker** [comment]
Resolved with PR #562
